modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -902,6 +902,19 @@ void ONNXImporter::populateNet(Net dstNet)
                 Mat bias = getBlob(node_proto, constBlobs, 2);
                 layerParams.blobs.push_back(bias);
             }
+            if (constBlobs.find(node_proto.input(0)) != constBlobs.end())
```

Comment:
Is that a configuration for GEMM over two constant inputs? Can we use `addConstant`?

Reply:
> Is that a configuration for GEMM over two constant inputs? Can we use `addConstant`?

As I see from code in fully_connected_layer.cpp, there are only 2 configurations for GEMM layer, with 1 input and with 2 inputs, so it assumes that there is at least 1 input layer, which is connected in the graph, not constant.